app/models/user.rb

```diff
@@ -66,6 +66,10 @@ def supervisor?
     permission.try(:name) == 'Supervisor'
   end
 
+  def system?
+    permission.try(:name) == 'System'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.